#! /bin/sh /usr/share/dpatch/dpatch-run
## Use system tiff headers.dpatch by Robin Cornelius <robin.cornelius@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use system tiff headers

@DPATCH@

Index: ./codec/convert.c
===================================================================
--- ./codec/convert.c	(revision 507)
+++ ./codec/convert.c	(working copy)
@@ -33,7 +33,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include "openjpeg.h"
-#include "../libs/libtiff/tiffio.h"
+#include <tiffio.h>
 #include "convert.h"
 
 /*
